### DISCUSSION TITLE
[bot]插件一直显示约束未满足

### DISCUSSION THREAD

**Discussion by Realzhazha** [AnzhiZhang/MCDReforgedPlugins#195] [Q&A] [answered]
如题，有些人可以使用!!bot save 1 之类的指令，但是有些人用就会一直显示约束未满足
![微信图片_20240627011337](https://github.com/AnzhiZhang/MCDReforgedPlugins/assets/35806568/bb481d42-876c-4664-87ca-27f41421e501)
控制台也没有信息
![image](https://github.com/AnzhiZhang/MCDReforgedPlugins/assets/35806568/4a54b190-1282-4358-9078-fcc90a853278)

**Comment by AnzhiZhang** [comment, marked as answer]
权限不足

https://github.com/AnzhiZhang/MCDReforgedPlugins/tree/master/src/bot#permissions
https://docs.mcdreforged.com/zh-cn/latest/permission.html
